src/app/rooms/admin/room-form/room-form.component.ts

```diff
@@ -73,6 +80,22 @@ export class RoomFormComponent implements OnInit {
       // Create empty form with the room model
       this.room = new Room();
     }
+
+  }
+
+  /**
```

Comment:
If you remove the old datepicker, these two function will need updating or be removed, see how you go